formulas/functions/statistical.js

```diff
@@ -215,7 +215,23 @@ const StatisticalFunctions = {
 
   MAX: () => {},
 
-  MAXA: () => {},
+  MAXA: (...values) => {
+    const arr = [];
+    arr.push(...values.map(a => a.value))
+    const flatArr = FormulaHelpers.flattenDeep(arr)
+    const updatedArr = flatArr.map(a => {
+      if(a === true)
+        return 1
+      if(!isNaN(parseFloat(a)))
+        return parseFloat(a)
+      if(!isNaN(parseInt(a)))
+        return parseInt(a)
+      return 0
+    })
+    if(updatedArr.length === 0)
+      return 0;
```

Comment:
This check could be done on `flatArr`,  before the map, since `flatArr` and `updatedArr` will always be the same length with that map function.  Also, I'm not sure if this should return `0` or throw an error (`N/A`?) in that case.

Reply:
Excel returns 0